commands/command_pull.go

```diff
@@ -113,7 +113,7 @@ func pull(filter *filepathfilter.Filter) {
 	}
 
 	if singleCheckout.Skip() {
-		fmt.Println(tr.Tr.Get("Skipping object checkout, Git LFS is not installed."))
+		fmt.Println(tr.Tr.Get("Skipping object checkout, Git LFS is not installed for this repository. Consider installing it with 'git lfs install'."))
```

Comment:
I think this text is clearly an improvement, but I think it's longer than 80 characters, which might make it a bit difficult to read in some cases.  Could we insert a newline here after `repository.` to make it easier to read on systems with small screens?

Reply:
I totally agree. I amended the PR with a multiline message. This makes it easier to read for both developers and end-users. However, I couldn't find an example of a multiline string in the project. If you prefer backticks, I can adjust it to that as well. Thank you for the warm welcome.